charts/kyverno/templates/deployment.yaml

```diff
@@ -30,8 +30,17 @@ spec:
       {{- with .Values.podSecurityContext }}
       securityContext: {{ tpl (toYaml .) $ | nindent 8 }}
       {{- end }}
-      {{- with .Values.affinity }}
-      affinity: {{ tpl (toYaml .) $ | nindent 8 }}
+      {{- with .Values.antiAffinity.enable }}
```

Comment:
I think this should be `if` and not `with`.

Reply:
okay. Made the changes.